docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -112,61 +114,107 @@ After the active instance has been configured, you can configure one or more sta
 
 Your standby instance is now configured.
 
+### Verify HA
+
+To confirm communication between the active and standby, you can do the following:
+
+- Click **Make Active** on the standby. You should see a list of available backups that you can restore from. (Don't promote the standby.)
```

Comment:
(Don't promote the standby.)- does it mean don't do it yet before next step.. I thought clicking make active would promote the standby. can you clarify?